Islandora/islandora_marcxml
# Update XSL options and add admin form select

**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-2404)

# What does this Pull Request do?

Adds the latest versions of the LoC MARC-to-MODS and MODS-to-MARC XSL files, with admin form select options so you can still use the old ones if you prefer them.

# What's new?

- New XSL files from official LOC source
- New admin form options so you can select them
- Default value updated to the latest version in XSLT 1.0 to minimize any potential (unlikely) conflicts

# How should this be tested?

- Get a sample MARCXML file (eg http://ebba.english.ucsb.edu/content/marcxml.html)
- Configure islandora_marcxml to use different versions of the XSLT each time you ingest an object using this file
- Confirm you've got valid data
- View an object's MARCXML tab and make sure that you are still getting valid MARC after changing MODS-to-MARC versions.

# Interested parties
@Islandora/7-x-1-x-committers
